src/components/IllustratedHeaderPageLayout.js

```diff
@@ -51,15 +54,18 @@ function IllustratedHeaderPageLayout({backgroundColor, children, illustration, f
                         iconFill={backgroundColor === themeColors.appBG ? undefined : themeColors.iconColorfulBackground}
                     />
                     <View style={[styles.flex1, StyleUtils.getBackgroundColorStyle(themeColors.appBG), !isOffline ? safeAreaPaddingBottomStyle : {}]}>
-                        {/* Safari on ios has a bug where overscrolling the page scrollview shows green the background color. This is a workaround to fix that. https://github.com/Expensify/App/issues/23422#issuecomment-1666890994 */}
-                        <View style={[styles.dualColorOverscrollSpacer]}>
-                            <View style={[styles.flex1, StyleUtils.getBackgroundColorStyle(backgroundColor)]} />
-                            <View style={[styles.flex1, StyleUtils.getBackgroundColorStyle(themeColors.appBG)]} />
-                        </View>
+                        {/* Safari on ios/mac has a bug where overscrolling the page scrollview shows green the background color. This is a workaround to fix that. https://github.com/Expensify/App/issues/23422 */}
+                        {Browser.isSafari() && (
```

Comment:
Made changes specifc to safari, on Android when there is only one workspace there is dual background visible.